bench/src/main/scala/geotrellis/raster/summary/PolygonalSummaryBench.scala

```diff
@@ -44,11 +45,21 @@ class PolygonalSummaryBench {
     multibandGeom  = multibandGeoTiff.extent.toPolygon
   }
 
+  // Bench the MeanVisitor because it uses a class instead of an AnyVal
```

Comment:
Turns out these new versions with the wrapped types are a bit slower. But not massively so:

![Screen Shot 2019-06-21 at 8 45 21 AM](https://user-images.githubusercontent.com/1818302/59932626-9960e900-9415-11e9-9819-9ba1b2b25df0.png)